components/UnitIntroduction.vue

```diff
@@ -1,7 +1,10 @@
 <template>
   <div>
     <h1>Analyze your private data</h1>
-    <p class="body-1">
+    <p v-if="isGenericViewer" class="body-1">
```

Comment:
We should have a more generic solution for adding a experience specific text, like taking it from a description field in the experience's json

Reply:
I understand that it might be cleaner to put this text in the manifests, but I don't know if it would be more generic